CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Rodio Development Guide
+
+## Quick Start
+
+1. Install [Rust](https://www.rust-lang.org/tools/install)
```

Comment:
- Clone the main branch
- Create a new branch for your contribution
- Build rodio using cargo build

Reply:
Think that should be in the Quick Start?

Reply:
I saw the later section just now. I would remove Install `Rust` though, I think its fair to assume potential contributors to a rust library have rust installed.

Reply:
Good point :+1: